src/v2/Components/RecentlyViewed.tsx

```diff
@@ -34,6 +35,7 @@ export class RecentlyViewed extends React.Component<RecentlyViewedProps> {
 
   render() {
     const { me } = this.props
+    const isMobile = sd?.IS_MOBILE
```

Comment:
Helpful when making layout changes like this to run the page with JS turned off (which simulates the SSR pass) and see how the layout works out there.